packages/workflow/src/RoutingNode.ts

```diff
@@ -236,11 +236,15 @@ export class RoutingNode {
 			destinationOptions.maxResults = sourceOptions.maxResults
 				? sourceOptions.maxResults
 				: destinationOptions.maxResults;
+
 			merge(destinationOptions.options, sourceOptions.options);
+
 			destinationOptions.preSend.push(...sourceOptions.preSend);
 			destinationOptions.postReceive.push(...sourceOptions.postReceive);
+
 			if (sourceOptions.requestOperations) {
 				destinationOptions.requestOperations = Object.assign(
+					{},
```

Comment:
(Added this as I would get a type incompatibility warning in the editor without)